File: RxAndroidRetrofit/app/src/main/java/com/example/rxandroidretrofit/Android.java
package com.example.rxandroidretrofit;

public class Android {
    private String ver;
    private String name;
    private String api;

    public Android(String ver, String name, String api) {
        this.ver = ver;
        this.name = name;
        this.api = api;
    }

    public Android(){

    }

    public String getVer() {
        return ver;
    }

    public void setVer(String ver) {
        this.ver = ver;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getApi() {
        return api;
    }

    public void setApi(String api) {
        this.api = api;
    }
}

File: demoRecyclerView/app/src/main/java/com/example/demorecyclerview/Android.kt
package com.example.demorecyclerview

class Android {
    //@SerializedName("ver")
    //@Expose
    private var ver: String? = null
    //@SerializedName("name")
    //@Expose
    private var name: String? = null
    //@SerializedName("api")
    //@Expose
    private var api: String? = null

    constructor(ver: String?, name: String?, api: String?) {
        this.ver = ver
        this.name = name
        this.api = api
    }


    fun getVer(): String? {
        return ver
    }

    fun setVer(ver: String) {
        this.ver = ver
    }

    fun getName(): String? {
        return name
    }

    fun setName(name: String) {
        this.name = name
    }

    fun getApi(): String? {
        return api
    }

    fun setApi(api: String) {
        this.api = api
    }
}